manifests/RProject/R/4.0.0.yaml

```diff
@@ -0,0 +1,12 @@
+Id: RProject.R
+Publisher: RProject
+Name: RProject R
+Version: 4.0.0
+License: GNU General Public License, version 2
+LicenseUrl: https://www.r-project.org/Licenses/
```

Comment:
There is also this licence file available
https://www.r-project.org/COPYING
I don't know which one should be included in the manifest

Reply:
I think the link you are referring is more suitable.